src/website/views/components/skeleton/_examples.pug

```diff
@@ -0,0 +1,157 @@
+h2 Examples
+
+h3 Base
+.example.-mb--3
+  .-p--3.-p-lg--6
+    .chi-skeleton
+  .example-tabs.-pl--2
+    ul.chi-tabs#examples-base
+      li.-disabled
+        a(href=`#webcomponent-base`) Web Component
+      li.-active
+        a(href=`#html-base`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-base
+  .chi-tabs-panel.-active#html-base
+    :code(lang="html")
+      <div class="chi-skeleton"></div>
+
+h3 Rounded
+.example.-mb--3
+  .-p--3.-p-lg--6
+    .chi-skeleton.-rounded
+  .example-tabs.-pl--2
+    ul.chi-tabs#examples-rounded
+      li.-disabled
+        a(href=`#webcomponent-rounded`) Web Component
+      li.-active
+        a(href=`#html-rounded`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-rounded
+  .chi-tabs-panel.-active#html-rounded
+    :code(lang="html")
+      <div class="chi-skeleton -rounded"></div>
+
+h3 Circle
+.example.-mb--3
+  .-p--3.-p-lg--6
+    .chi-skeleton.-circle
+  .example-tabs.-pl--2
+    ul.chi-tabs#examples-circle
+      li.-disabled
+        a(href=`#webcomponent-circle`) Web Component
+      li.-active
+        a(href=`#html-circle`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-circle
+  .chi-tabs-panel.-active#html-circle
+    :code(lang="html")
+      <div class="chi-skeleton -circle"></div>
+
+h3 Square
+.example.-mb--3
+  .-p--3.-p-lg--6
+    .chi-skeleton.-square
+  .example-tabs.-pl--2
+    ul.chi-tabs#examples-square
+      li.-disabled
+        a(href=`#webcomponent-square`) Web Component
+      li.-active
+        a(href=`#html-square`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-square
+  .chi-tabs-panel.-active#html-square
+    :code(lang="html")
+      <div class="chi-skeleton -square"></div>
+
+h3 Sizes
```

Comment:
@janibolkvadze please change example sizes to be consisten with documentation order (i.e: base, rounded, circle and square)